docs/content/latest/reference/configuration/yb-tserver.md

```diff
@@ -485,6 +485,16 @@ For details on how online index backfill works, see the [Online Index Backfill](
 
 Default: `true`
 
+##### --ysql_sequence_cache_minval
+
+Specify how many numbers from the sequence to cache in the client for every sequence object.
```

Comment:
Should this say something like `Specify the minimum number of sequence values to cache ...`?

Reply:
Makes sense since it is not always guaranteed this value will be used as the cache size if cache clause has greater value. 
Made change as specified.